### DISCUSSION TITLE
rel selector doesn't work

### DISCUSSION THREAD

**Discussion by Artiom-M** [biati-digital/glightbox#414] [Q&A] [answered]
I have a selector like this:
```js
const lightbox = GLightbox({"selector":"[rel=\"ligthbox\"]"});
```
And image tag:
```html
<a rel="lightbox" data-width="1200" data-height="1200" href="/images/7/0 0....
```
It works with class style selector.

Also another question, how to put 2 or more selectors in the config?

**Comment by gingerchew** [comment, marked as answer]
In your JS example, you spelled lightbox wrong. The selector you pass in only goes to a `.querySelector()` function.

**Reply by Artiom-M** [reply]
Thank you. That was exactly it!
